### DISCUSSION TITLE
Organization Permissions in Admin Console and Web Vault

### DISCUSSION THREAD

**Discussion by Gerardv514** [dani-garcia/vaultwarden#4731] [Q&A]
Running the latest VW version. I have an organization in which I am an Admin of. There are 2 collections, collection 1 is set to No Access and collection 2 is set to Can edit.

On BW ios apps, web vault, and browser extension this works fine where I cannot see any items of collection 1 since I have no access. I can see and edit collection 2.

The issue (possibly if this is an issue or intended): On the Web Vault, When I go into the admin console, then vault, I see the two collections one says no access and the 2nd says can edit. If I click into collection 1 I can see all items in that collection including the passwords. With the No Access role, shouldn't I just be able to manage the collection and not see the individual items?

**Comment by BlackDex** [comment]
If you are an admin and view it via the admin console, then you are allowed to see everything.

If you view it via your personal console you will not see them.

Also, this is a client-side item, and not something which can be controlled by the server backend. The server sends all vault data in one go.

Vaultwarden doesn't modify the actual working of the web-vault client except for making sure all features works and name changes.

**Reply by Gerardv514** [reply]
Understood this is not a Vaultwarden thing, just looking to clarify how I'm reading the documentation then. So to confirm, as an admin and when in the admin console with no access to a specific collection, an admin should be able to see everything as in the collection AND the individual items in that collection? I would of though an admin could see the collection to manage it, but not the individual items.

**Reply by BlackDex** [reply]
The admin and owner can see everything. Only a manager (or a can manage user) can only manage them.
